### PR TITLE
Exclude environment files from GitHub

### PR DESCRIPTION
**Pull request by scriptPilot**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ services:
     volumes:
       - "${DOCKER_DATA}/bricks:/database"
     env_file:
-      - sample.env
+      - .env
```
Comment (brighteyed) [inline docker-compose.yml:10]: I think that's inconsistent to have `sample.env` on disk and `.env` file in `docker-compose.yml`

Comment (scriptPilot) [reply docker-compose.yml:10]: Please refer to the changes in the README file.
The trick is to avoid that any credentials are exposed to the repository.

Comment (brighteyed) [reply docker-compose.yml:10]: I agree. It is better to prevent potentially leaked credentials

**Review by brighteyed — APPROVED** [review]

**Comment by scriptPilot** [comment]
Thanks for the acceptance :-)

**Pull request merged**